docker/build-common.sh

```diff
@@ -7,7 +7,12 @@ set -e
 # It unifies RUSFLAGS, compilation flags (like --release) and set of binary crates to compile in common docker layer
 
 export RUSTFLAGS="-Ctarget-cpu=skylake -Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3"
-cargo build --release -p libra-genesis-tool -p libra-node -p cli -p config-builder -p libra-key-manager -p safety-rules -p db-bootstrapper -p backup-cli -p cluster-test "$@"
+
```

Comment:
Some folks may get confused on the `RUST_NIGHTLY` and interpreting as we are building the binaries with the nightly Rust compiler toolchain. Can we put a note for clarification?